test/passing/source.ml.ref

```diff
@@ -8186,8 +8186,8 @@ module USERCODE (X : TYPEVIEW) = struct
   module type F = functor (C : CORE with type V.usert = X.combined) -> BARECODE
```

Comment:
A separate issue for later, but it looks like we're missing a break hint after the `->`

Reply:
Before or after

Reply:
I assumed after, but I don't have large body of code using such explicit functors to get a good feel for what works best.